src/components/charts/linechart.js

```diff
@@ -164,12 +164,12 @@ export default {
                 fontFamily: 'Open Sans'
               },
               time: {
-                unit: 'day',
+                unit: this.$parent.$store.getters[this.$parent.path + '/intervalUnit'],
                 unitStepSize: 15,
                 displayFormats: {
                   day: 'M/DD',
                   hour: 'dd h:mm a',
-                  minute: 'h:mm a'
+                  minute: 'dd h:mm a'
```

Comment:
Not sure if day of the week or day of the month is better, but if we want to use day of the month here, I guess change this to `M/DD h:mm a`

See also: https://www.chartjs.org/docs/latest/axes/cartesian/time.html#display-formats